Sources/HaCWebsiteLib/Markdown+relativeURLs.swift

```diff
@@ -0,0 +1,48 @@
+import Regex
+import Foundation
+
+extension Markdown {
+  /*
+   * Markdown references (links and images) can contain relative URLs
+   * This method will resolve relative URLs relative to a given base
+   * e.g. '[Meow](images/cat.png)' -> '[Meow](https://hackersatcambridge.com/images/cat.png)'
+   */ 
+  mutating func resolveRelativeURLs(relativeTo baseURL: URL) {
```

Comment:
Why does this have to be mutating? Could it be one of

- Exposed as a string-to-string transformation on markdown sources
- A function returning a new `Markdown` with a different source (`withRelativeURLsResolved`)

Reply:
Went for the latter since doing this as a string-to-string transformation only makes sense in the case of Markdown sources